pkg/utils/json.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"openpitrix.io/openpitrix/pkg/logger"
+)
+
+func Loads(src string, dest interface{}) error {
```

Comment:
loads is not a good name in this package, maybe loadsxxx is better

Reply:
It just keep the same with standard json lib we used in python. 
http://www.runoob.com/python/python-json.html